From 2f476df340f50b9c10a7f45f4ec809e9ef82aba6 Mon Sep 17 00:00:00 2001
From: Thomas Petazzoni <thomas.petazzoni@free-electrons.com>
Date: Tue, 2 Sep 2014 22:40:04 +0200
Subject: [PATCH 2/4] Avoid libtool/AM_PROG_AR warning during autoreconf

This patch allows to avoid the following warning during autoreconf:

/home/thomas/projets/buildroot/output/host/usr/share/automake-1.14/am/ltlibrary.am: warning: 'lib/libiscsi_convenience.la': linking libtool libraries using a non-POSIX
/home/thomas/projets/buildroot/output/host/usr/share/automake-1.14/am/ltlibrary.am: archiver requires 'AM_PROG_AR' in 'configure.ac'

Note that LT_INIT *must* be after AM_PROG_AR, otherwise:

configure.ac:14: warning: LT_INIT was called before AM_PROG_AR

Signed-off-by: Thomas Petazzoni <thomas.petazzoni@free-electrons.com>
---
 configure.ac | 4 +++-
 1 file changed, 3 insertions(+), 1 deletion(-)

diff --git a/configure.ac b/configure.ac
index 6d29af0..eb9d995 100644
--- a/configure.ac
+++ b/configure.ac
@@ -5,12 +5,14 @@ AC_CONFIG_MACRO_DIR([m4])
 
 AM_INIT_AUTOMAKE([-Wall -Werror foreign subdir-objects])
 AM_SILENT_RULES
-LT_INIT
 
 AC_CANONICAL_HOST
 AM_CONDITIONAL(LD_ISCSI,
                [expr "$host_os" : linux > /dev/null 2>&1])
 AM_PROG_CC_C_O
+AM_PROG_AR
+
+LT_INIT
 
 enable_write_strings="yes"
 
-- 
2.0.0

